src/tlo/methods/hiv.py

```diff
@@ -2534,13 +2541,15 @@ def do_at_initiation(self, person_id):
         df = self.sim.population.props
         person = df.loc[person_id]
 
-        # Check if drugs are available, and provide drugs:
+        # Check if drugs are available, and provide drugs
```

Comment:
It would be safer for this to be dict or named tuple so that we don't have to remember the special ordering. 
Ie for example return
{"art": True, "contrim": False}

Reply:
I've changed this